deeppavlov/core/agent_v2/state_schema.py

```diff
@@ -83,6 +84,7 @@ def to_dict(self):
             'active_skill': self.active_skill,
             'confidence': self.confidence,
             'text': self.text,
+            'sent_text': self.sent_text or self.text,
```

Comment:
`or` logic should be included in postprocessor, in this method we only serialize